website/blog/2024-1-18-while-youre-busy-fixing-vulnerabilities/item.md

```diff
@@ -0,0 +1,115 @@
+
+You can also set up alerts: these will send an email to the security team in the event that a new role is created so you can go and investigate.
+
+Full disclosure: we make (and use in-house) [authentik](https://goauthentik.io/), which is an identity provider. There is an open source version, so even if you’re strapped for cash you can still implement IAM. You can [check if authentik integrates with your application here](https://goauthentik.io/integrations/).
+
+## What else should you be doing (at least) quarterly?
+
+While we’re on the topic of security hygiene, there are other practices that are worth doing routinely (especially if you’ve automated your access management with an identity provider).
+
+If you’re on a path to a certification like SOC II or ISO/IEC 27001, you will already be implementing these on a quarterly basis. But as we’ve discussed on this blog before, [even if you’re not seeking certification, they’re still worth doing](https://goauthentik.io/blog/2023-11-22-how-we-saved-over-100k#do-you-really-need-certifications) to ingrain good practices that become more important as you scale. As the Redditor suggested above, [security then goes from reactionary to policy making](https://www.reddit.com/r/cybersecurity/comments/12ygfnw/comment/jhnqgnt/).
+
+-   **Review and rotate long-lived credentials:** What was the last time AWS access keys were rotated? Rotating credentials every 30/60/90 days contributes to [“defense in depth”, which we touched on before here](https://goauthentik.io/blog/2023-11-22-how-we-saved-over-100k#organizational-security)). That way, if an access key is compromised, it’s no longer valid. Ideally, you would automate rotation so you can set and forget about this one.
+-   **Identify gaps in coverage:** A lot of these routine activities have a second-order effect of exposing failures in process. You’re kicking over stones and seeing what’s lurking underneath. Anything you find is signal that there’s a gap in coverage. How do you make sure you catch those long-lived tokens without having to go looking for them? Apart from automating rotation of credentials as suggested above, with AWS, for example, you could write a simple Python script to query IAM and send out an email when access tokens exceed a certain threshold.
+
+### Designate ownership
+
+If you’re a small company and don’t have dedicated security professionals to “own” your apps’ and services’ security, share the responsibilities. For example, one team member owns one application hosted on these five servers, and is responsible for them:
+
+-   They perform the routine hygiene checks (like access reviews)
+-   They also have the context necessary in the event of a critical vulnerability or even an outage (beyond security, you don’t want people having to run around trying to figure out who does what or how to fix things in an emergency)
```

Comment:
Suggestion: Add a clause about "They're responsible for writing the [runbook](https://www.techtarget.com/searchnetworking/definition/run-book) and updating its procedures after an outage or vulnerability has been identified and managed."